api/src/main/java/org/openmrs/util/UpgradeUtil.java

```diff
@@ -24,11 +25,11 @@
 import org.openmrs.api.APIException;
 
 public class UpgradeUtil {
-	
```

Comment:
@MalshaL can you please double check your formatter settings am wondering why there are so many whitespace formatting changes in this file, I dont get them with my eclipse config (https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE).